smithy-cli/src/main/java/software/amazon/smithy/cli/commands/ValidationEventFormatOptions.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package software.amazon.smithy.cli.commands;
+
+import java.util.function.Consumer;
+import software.amazon.smithy.cli.ArgumentReceiver;
+import software.amazon.smithy.cli.CliError;
+import software.amazon.smithy.cli.CliPrinter;
+import software.amazon.smithy.cli.HelpPrinter;
+import software.amazon.smithy.model.shapes.ShapeId;
+import software.amazon.smithy.model.validation.ValidationEvent;
+import software.amazon.smithy.model.validation.ValidationEventFormatter;
+
+final class ValidationEventFormatOptions implements ArgumentReceiver {
+
+    private static final String VALIDATION_FORMAT = "--validation-format";
+
+    enum ValidationFormat {
+        TEXT {
+            @Override
+            void print(CliPrinter printer, ValidationEventFormatter formatter, ValidationEvent event) {
+                printer.println(formatter.format(event));
+            }
+        },
+
+        CSV {
+            @Override
+            void beginPrinting(CliPrinter printer) {
+                printer.println("severity,id,shape,file,message,hint,suppressionReason");
```

Comment:
The `file` field also includes the line number so maybe `location` would be clearer

Reply:
Hmmm... since I don't fully agree here, I think I'll just go ahead and split this into file, line, column. Checkstyle does this in their reports too.